wca-integration/src/main/java/eu/itesla_project/wca/report/WCAReportImpl.java

```diff
@@ -408,9 +408,9 @@ private void exportPostContingencyViolations(Path folder) throws IOException {
         };
         CsvTableFormatterFactory factory = new CsvTableFormatterFactory();
         try (Writer writer1 = Files.newBufferedWriter(violationsPath1, StandardCharsets.UTF_8);
-             TableFormatter formatter1 = factory.create(writer1, POST_CONTINGENCY_VIOLATIONS_WITHOUT_UNCERTAINTIES_TITLE, TABLE_FORMATTER_CONFIG, COLUMNS);
+             TableFormatter formatter1 = factory.create(writer1, POST_CONTINGENCY_VIOLATIONS_WITHOUT_UNCERTAINTIES_TITLE, TABLE_FORMATTER_CONFIG, columns);
```

Comment:
Fix indentation

Reply:
Indentation should be 4 spaces. I think this block is using 5 spaces. You have to change that to respect the coding style

Reply:
https://stackoverflow.com/questions/30753996/what-is-the-code-convention-for-formatting-try-with-resources
And the idea refused to back one space when i typed "back". When i typed back, this line will go to the end of last line